.github/workflows/tests.yml

```
@@ -41,7 +41,7 @@ jobs:
         stable: [false, true]
     name: tap_syntax${{ matrix.stable && ' (stable)' || '' }}
     runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.stable }}
+    continue-on-error: ${{ matrix.stable || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'CI-no-fail-fast-syntax')) }}
```

```suggestion
    continue-on-error: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'CI-no-fail-fast-syntax') }}
```
may be worth doing this which would mean you'd need it for stable syntax errors too?